pkg/destroy/vsphere/vsphere.go

```diff
@@ -43,6 +42,7 @@ type ClusterUninstaller struct {
 
 	Logger logrus.FieldLogger
 
+	//nolint:all
 	context context.Context
```

Comment:
We shouldn't be storing `context` inside the struct. See https://github.com/openshift/installer/pull/6604/commits/702f869894c1922b0c6b6552dea768418655bf8e

Reply:
@r4f4 I will pick up your changes after #6604 merges